### PR TITLE
Fix namespace

### PR DESCRIPTION
**Pull request by ril1**
This PR is to fix issue `read_geospatial_file() not yet included in package but is in namespace`

### CODE REVIEW

**Comment by ril1** [comment]
@mlhollestelle can you documentation for the `get_file_ext` function?

**Review by mlhollestelle — DISMISSED** [review]
I made sure the new functions that had creeped up in the namespace are actually part of the package now. @ril1 , could you please have a quick review before merge? Thanks.

File: R/geospatial.R [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    if(file_ext == ".shp") {
+        geospatial_file = read.shape(new_connection)
+    }
+    if(file_ext == ".gpkg") {
+        geospatial_file = read.gpkg(new_connection)
+    }
+    if(file_ext == ".TAB") {
+        geospatial_file = read.mapinfo(new_connection)
+    }
+    return(geospatial_file)
```
Comment (ril1) [inline R/geospatial.R:22]: Would it be better if all these `if` statements were `else if` statements? and perhaps directly under that would just be `return(read.shape(new_connection))`. Any thoughts @mlhollestelle?

Comment (mlhollestelle) [reply R/geospatial.R]: Good idea, done. There were actually still some other mistakes in the function which I now solved as well.

**Review by mlhollestelle — APPROVED** [review]

**Pull request merged**